app/workers/lighthouse/document_upload.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require 'ddtrace'
+require 'timeout'
+
+class Lighthouse::DocumentUpload
+  include Sidekiq::Worker
+
+  # retry for one day
+  sidekiq_options retry: 14, queue: 'low'
+  # Set minimum retry time to ~1 hour
+  sidekiq_retry_in do |count, _exception|
+    rand(3600..3660) if count < 9
+  end
+
+  def perform(_auth_headers, user_icn, document_hash)
```

Comment:
Can you explain what the `_auth_headers` parameter is here for?

Reply:
Ah that was how EVSS was uploading documents. From what I could tell, we only needed the user_icn so _auth_headers ended up going unused. Removed